### DISCUSSION TITLE
[Script request] Kimai

### DISCUSSION THREAD

**Discussion by florianwilde** [community-scripts/ProxmoxVE#107] [Request script]
### Application Name

Kimai OpenSource Timerecording

### Website

https://www.kimai.org/de/

### Description

Kimai is an open-source time-tracking software designed for freelancers, small teams, and businesses to efficiently track, manage, and analyze work hours. This web-based tool enables users to log their work time with ease, associating entries with specific clients, projects, and tasks.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
#Blocked

In Pipeline by me.

**Reply by Smile2Grow** [reply]
You can install it meanwhile with runtipi
